marketplaceItems/avatarCustomization101/README.md

```diff
@@ -238,4 +238,9 @@ This app showcases material entities, blendshapes, and flow to customize your av
 # Releases
 
 ## 2019-03-05_09-11-00 :: [a9cb28d](https://github.com/highfidelity/hifi-content/commit/a9cb28d)
-- Initial release
\ No newline at end of file
+- Initial release
+
+## 2019-07-09_02-20-00 :: [68106c9](https://github.com/highfidelity/hifi-content/commit/68106c9)
```

Comment:
New releases go at the top of the release notes; please swap the order of these two entries.